app/views/records/edit_fields/_chronology_note.html.erb

```diff
@@ -0,0 +1,19 @@
+<%# Copied from Hyrax to support dynamic labels and hints for custom worktypes %>
+
+<%# Avoid NoMethod error when rendering partial in Collection form %>
```

Comment:
We aren't doing custom labels per worktype so you can just use a simple text area form:
```
<% if f.object.multiple? key %>
  <%= f.input :chronology_note, as: :multi_value, input_html: { rows: '14', type: 'textarea'}, required: f.object.required?(key) %>
<% else %>
  <%= f.input :achronology_note, as: :text, input_html: { rows: '14' }, required: f.object.required?(key) %>
<% end %>
```